otter/cloud_client/rcv3.py

```diff
@@ -189,7 +191,9 @@ def _check_bulk_add(attempted_pairs, result):
     if errors:
         raise BulkErrors(errors)
     elif exists:
-        to_retry = pset(attempted_pairs) - exists
+        # lb_id is case-insensitive and could've been called with upper case
+        attempted = pset(map(lambda p: (p[0].lower(), p[1]), attempted_pairs))
```

Comment:
`map` is sort of softly deprecated (it's not going away but it's considered a stylistic faux pas); you should generally use comprehensions.  Assuming that `pset` consumes its argument, you can just use a generator, like so: 

```python
        attempted = pset(p[0].lower(), p[1]) for p in attempted_pairs)
```